src/System.IO.FileSystem.Watcher.Polling/System/IO/PollingFileSystemWatcher.cs

```diff
@@ -49,8 +51,10 @@ public PollingFileSystemWatcher(string path, string filter = "*.*", EnumerationO
 
         public void Start()
         {
+            if (_disposed) throw new ObjectDisposedException(nameof(_timer));
```

Comment:
Should be `nameof(PollingFileSystemWatcher)`.

Reply:
@MisinformedDNA, after the CI goes green again, can you submit a commit for this and we can merge this in (barring any other feedback from @JeremyKuhne).